msal/tests/dev apps/DesktopTestApp/MsalUserRefreshTokenControl.cs

```diff
@@ -33,14 +35,16 @@ public MsalUserRefreshTokenControl()
 
         private void InvalidateRefreshTokenBtn_Click(object sender, System.EventArgs e)
         {
-            _item.RefreshToken = GarbageRtValue;
-            _cache.SaveRefreshTokenCacheItem(_item);
+            _rtItem.Secret = GarbageRtValue;
+            _cache.SaveRefreshTokenCacheItem(_rtItem, null);
```

Comment:
>ull); [](start = 55, length = 5)

you have lots of these... consider having an override that doesn't require this to be specified instead.